module/icp/algs/impl/impl.c

```diff
@@ -0,0 +1,175 @@
+
```

Comment:
Add CDDL header.

This is based off of `aes_impl.c`

What do I put for the Copyright line?

Reply:
Added.